sources/packages/api/src/route-controllers/student/model/student-sync-info.dto.ts

```diff
@@ -0,0 +1,7 @@
+export default class StudentSyncInfoDto {
```

Comment:
Added to keep the behavior already in place for sync user return this data.
It is a new class only because it was moved from User models to Student models.